core/src/test/scala/com/pingcap/tispark/datasource/BaseDataSourceTest.scala

```diff
@@ -37,6 +37,7 @@ class BaseDataSourceTest(val table: String,
   protected def tidbWrite(rows: List[Row],
                           schema: StructType,
                           param: Option[Map[String, String]] = None): Unit = {
+    refreshConnections()
```

Comment:
better to add `clear catalog cache` after each `drop table`

Reply:
DONE